src/ccl_background.c

```diff
@@ -61,17 +133,17 @@ double ccl_omega_x(ccl_cosmology * cosmo, double a, ccl_omega_x_label label, int
 	
   switch(label) {
   case ccl_omega_m_label :
-    return cosmo->params.Omega_m/(cosmo->params.Omega_m+cosmo->params.Omega_l*pow(a,-3*(cosmo->params.w0+cosmo->params.wa))*exp(3*cosmo->params.wa*(a-1))+cosmo->params.Omega_k*a+ (cosmo->params.Omega_g + cosmo->params.Omega_n_rel)/a + OmNuh2*a*a*a / (cosmo->params.h) / (cosmo->params.h));
+    return cosmo->params.Omega_m/(a*a*a) / h_over_h0(a, cosmo, status) / h_over_h0(a, cosmo, status);
```

Comment:
What's happening here? Having to evaluate h_over_h0 twice seems very inefficient (and I'm not sure the complier will be smart enough to optimize it automatically). It'd be better to do
```
#At the beginning of this function
hnorm=h_over_h0(a,cosmo,status)
...
#Then, later on
return cosmo->params.Omega_m/(a*a*a) /(hnorm*hnorm);
```